### DISCUSSION TITLE
Welcome to intl_phone_number_input Discussions!

### DISCUSSION THREAD

**Discussion by natintosh** [natintosh/intl-phone-number-input#145] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by prasun23** [comment]
Hi. Thank you very much much for providing such a great package. I am facing a challenge in using "InternationalPhoneNumberInput". It would be really helpful if you kindly guide me. I am trying to use this widget inside a form but when I am using _formKey.currentState.save(), I am getting null value. How to get the phone number saved in a variable?
